services/w3c/w3c-validation.tester.js

```diff
@@ -0,0 +1,100 @@
+'use strict'
+const Joi = require('@hapi/joi')
+const t = (module.exports = require('../tester').createServiceTester())
+
+const isErrorOnly = Joi.string().regex(/^[0-9]+ errors?$/)
+
+const isWarningOnly = Joi.string().regex(/^[0-9]+ warnings?$/)
+
+const isErrorAndWarning = Joi.string().regex(
+  /^[0-9]+ errors?, [0-9]+ warnings?$/
+)
+
+const isW3CMessage = Joi.alternatives().try(
+  'validated',
+  isErrorOnly,
+  isWarningOnly,
+  isErrorAndWarning
+)
+const isW3CColors = Joi.alternatives().try('brightgreen', 'red', 'yellow')
+t.create(
+  'W3C Validation page conforms to standards with no preset and parser with brightgreen badge'
+)
+  .get(
+    '/default.json?targetUrl=https://hsivonen.com/test/moz/messages-types/no-message.html'
```

Comment:
By the way where did you find these pages? Seems like they were setup explicitly to be used as test targets

Reply:
They were actually test pages that was provided in the documentation of the validator project
https://github.com/validator/validator/wiki/Service-%C2%BB-HTTP-interface